controllers/csvController.go

```diff
@@ -0,0 +1,44 @@
+package controllers
+
+import (
+	"encoding/json"
+	"github.com/LuisMG96/academy-go-q42021/common"
+	"github.com/LuisMG96/academy-go-q42021/services"
```

Comment:
For best practices, you can separate the core packages, your packages and third packages like following:

Go core

Project Local 

Third-Party